pkg/resources/syslogng/configcheck.go

```diff
@@ -266,3 +267,12 @@ func (r *Reconciler) newCheckPod(hashKey string) (*corev1.Pod, error) {
 func configCheckResourceName(hash string) string {
 	return fmt.Sprintf("syslog-ng-configcheck-%s", hash)
 }
+
+func (r *Reconciler) configCheckPodObjectMeta(name, component string) metav1.ObjectMeta {
+	objectMeta := r.SyslogNGObjectMeta(name, component)
+
+	// Ensure that no istio sidecar is injected by default. See: https://github.com/istio/istio/issues/6324
```

Comment:
Could you please add this to the PR description as well? Just so if someone clicks on the PR they will see it immediately without going through the changes.

Reply:
adding it now